provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -1435,6 +1444,55 @@
         "tagName",
         "revision"
       ]
+    },
+    "pulumiservice:index:TemplateSource": {
+      "description": "A source for Pulumi templates",
+      "properties": {
+        "organizationName": {
+          "description": "Organization name.",
+          "type": "string"
+        },
+        "sourceName": {
+          "description": "Source name.",
+          "type": "string"
+        },
+        "sourceURL": {
+          "description": "Github URL of the repository from which to grab templates.",
+          "type": "string"
+        },
+        "destination": {
+          "description": "Settings that get filled in on new project creation.",
+          "$ref": "#/types/pulumiservice:index:TemplateSourceDestination"
+        }
+      },
+      "required": [
+        "organizationName",
+        "sourceName",
+        "sourceURL"
+      ],
+      "inputProperties": {
+        "organizationName": {
+          "description": "Organization name.",
+          "type": "string"
+        },
+        "sourceName": {
+          "description": "Source name.",
+          "type": "string"
+        },
+        "sourceURL": {
+          "description": "Github URL of the repository from which to grab templates.",
+          "type": "string"
+        },
+        "destination": {
+          "description": "Settings that get filled in on new project creation.",
```

Comment:
```suggestion
          "description": "The default destination for projects using templates from this source.",
```

Reply:
Thank you, I was struggling to come up with a good description here!